algorithms/merging/merge.py

```diff
@@ -217,14 +217,23 @@ def merge_and_truncate(params, experiments, reflections):
 
     if params.reporting.wilson_stats:
         if not intensities.space_group().is_centric():
-            wilson_scaling = data_statistics.wilson_scaling(
-                miller_array=intensities, n_residues=params.n_residues
-            )  # XXX default n_residues?
-            # Divert output through logger - do with StringIO rather than
-            # info_handle else get way too much whitespace in output.
-            out = StringIO()
-            wilson_scaling.show(out=out)
-            logger.info(out.getvalue())
+            try:
+                wilson_scaling = data_statistics.wilson_scaling(
+                    miller_array=intensities, n_residues=params.n_residues
+                )  # XXX default n_residues?
+            except (IndexError, RuntimeError):
+                logger.error(
+                    "\n"
+                    "Error encountered during Wilson statistics calculation:\n"
+                    "Perhaps there are too few unique reflections.\n",
```

Comment:
may want to include `str(e)` in message?

Reply:
Done.